apps/docs/components/Sidebar/sidebar.css

```diff
@@ -0,0 +1,132 @@
+.hd-sidebar {
+    border-inline-end: 1px solid var(--hd-color-border-primary-weak);
+    position: sticky;
+    top: var(--hd-space-8);
+    height: calc(100vh - var(--hd-space-8));
+    min-width: 18rem;
+    max-width: 18rem;
+}
+
+.hd-sidebar__container {
+    padding-inline: var(--hd-space-3);
+    overflow-y: scroll;
+    scroll-margin-top: var(--hd-space-8);
+    overscroll-behavior: contain;
+    height: calc(100% - var(--hd-space-1));
+    width: calc(100% - var(--hd-space-1));
+    padding-block: var(--hd-space-8);
+    position: relative;
+    top: var(--hd-space-05);
+    user-select: none;
+}
+
+.hd-sidebar__list {
+    list-style-type: none;
+    display: flex;
+    flex-direction: column;
+}
+
+.hd-sidebar__list-item {
+    display: flex;
+    font-weight: 600;
+    margin-block-end: var(--hd-space-05);
+}
+
+.hd-sidebar__list-item a {
```

Comment:
You should avoid pointing to a tag, it is better to add a class to the `a` for example `.hd-sidebar__link`.

Reply:
I thought that since there's no way a link won't be anything else than a <a> that it could remove some burden from the DOM, I will change this.